### ISSUE TITLE
Proxy resolution using PAC file is not working as expected in 3.6 selenium grid.

### ISSUE THREAD

**Issue by shankarkc** [SeleniumHQ/selenium#4883]
## Meta -
OS:  
Linux

Selenium Version:  
3.6

Browser:  
chrome

Browser Version:  
61.0.3163.100 (64 bit)

## Expected Behavior -
When used with PAC file proxy we should be able to open the URL

## Actual Behavior -
Page opening timesout

## Steps to reproduce -
webdriver code used as shown below
```
Proxy proxy = new Proxy();
proxy.setProxyAutoconfigUrl("http://HEREGOESMYPROXYNAME:8083/accelerated_pac_base.pac");
caps.setCapability(CapabilityType.PROXY, proxy);
```

If I set the same proxy in my browser it opens the page fine without any issue. But when I use it selenium grid, selenium grid page timesout. This is internal site. So I think if proxy autoconfiguration is not handled properly it might be resulting using proxy and timingout. I dont suspect the PAC file because if i set the same autoconfiguration URL in IE/Chrome it works fine and opens the URL. Please let me know what other information you need to debug further. I will provide the details. **I have observed this issue after upgrading to 3.6.0** If I rollback to older grid version 3.0.1 I dont notice this issue.
Here are the screenshots
![proxyissue-notworkingonseleniumgrid](https://user-images.githubusercontent.com/4651863/31645600-9b10e1d4-b31a-11e7-82ac-12fed84984c4.jpg)
![proxyissue-workingonrealbrowser](https://user-images.githubusercontent.com/4651863/31645601-9b478df6-b31a-11e7-983a-b9576b391491.jpg)

If you have any suggestions to workaround this i really appreciate it. Please let me know if you need any information. I will update the defect with required information. One more thing I have observed is same URL opens in some other sessions fine. I have a test with setup and teardown. Same URL opens in setup and fails in teardown. Its kind of intermittent. 
Thanks
Shankar KC

**Comment by barancev** [comment]
For issues please provide a [concise reproducible test case](http://sscce.org/) and describe what results you are seeing and what results you expect.

See [CONTRIBUTING.md](https://github.com/SeleniumHQ/selenium/blob/master/CONTRIBUTING.md#issue-contributions)

**Comment by shankarkc** [comment]
Hi I found the root cause of the issue. As per selenium guidelines we need to supply PAC file URL. IT folks make actual pac file not accessible directly ie; http://proxy:8083/accelerated_pac_base.pac was not returning the pac file. But http://proxy:8083 was returning the pac file. Once I changed it to http://proxy:8083 it started working. But I am still not sure why/How its working with old selenium grid. As i am not blocked anymore I am closing the issue.
